README.md

```diff
@@ -381,12 +381,106 @@ rer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" https://metrics.
+This section describes additional guidelines, conventions, and the overall
+process for writing release notes.
+
+#### Guidelines
+
+* Each release should contain release notes
+* Changes should be applicable to at least one of the six types listed below
+* Use literals for code and configuration
+  - ``defaultScanSettingsSchedule``
+  - ``nodeSelector``
+* Write your notes with users as the audience
+* Keep notes as short as possible and link to additional documentation when necessary
+  - Bug fixes should link to bug reports (GitHub Issues or Jira items)
+  - Features or enhancements should link to formal RFEs (GitHub Issues or Jira
+    items)
+
+#### Change Types
+
+The following describe each potential section for a release changelog.
+
+1. Enhancements
+2. Fixes
+3. Internal Changes
+4. Deprecations
+5. Removals
+6. Security
+
+*Enhancements* are reserved for communicating any new features or
+functionality. You should include any new configuration or processes a user
+needs to take to use the new feature.
+
+*Fixes* are for noting improvements to any existing functionality.
+
+*Internal Changes* are ideal for communicating refactors not exposed to end
+users. Even if a change does not directly impact end users, it is still
+important to highlight paying down technical debt and the rationale for those
+changes, especially since they impact the project's roadmap.
+
+*Deprecations* is for any functionality, feature, or configuration that is
+being deprecated and staged for removal. Deprecations should include why we're
+preparing to remove the functionality and signal any suitable replacements
+users should adopt.
+
+*Removals* is for any functionality, feature, or configuration that is being
+removed. Typically, entries in this section will have been deprecated for some
+period of time.
+
+*Security* is reserved for any security-related change, or relevant to a
+security-focused audience. Readers focused on only the security-specific
+content for a release should be able to glean all the information from this
+section.
+
+A change can apply multiple change types. For example, a bug fix for a security
+issue should be mentioned in the *Fixes* and *Security* sections. Another
+example would be a feature that improves the default security posture of the
+project should be mentioned in the *Enhancements* and *Security* sections.
+
+#### Process
+
+Contributors must include a release note with their changes. New notes should
+be added to the [Unreleased section](CHANGELOG.md#unreleased) of the
+[changelog](CHANGELOG.md]. Reviewers will assess the accuracy of the release
+note against the change.
+
+Maintainers preparing a new release will propose a change that renames the
+[Unreleased release notes](CHANGELOG.md#unreleased) to the newly released
+version and release date. Maintainers can remove empty sections if it does not
+contain any release notes for a specific release.
```

Comment:
remove empty sections or explicitly call out none?

Reply:
Looking at some of the existing release notes, we leave out sections that are not relevant.

I'm OK with either approach though.